docs/en/ingest-management/fleet/fleet-server-secrets.asciidoc

```diff
@@ -0,0 +1,265 @@
++
+If you are <<elastic-agent-container,running {fleet-server} under {agent} in a container>>, the environment variable `FLEET_SERVER_CERT_KEY_PASSPHRASE` may be used to specify the file path.
++
+APM API Key::
+The APM API Key may be used to gather APM data from {fleet-server}.
++
+It can be specified directly in the instrumentation segment of the configuration:
++
+[source,yaml]
+----
+inputs:
+  - type: fleet-server
+    instrumentation.api_key: my-apm-api-key
+----
++
+Or by a file with:
++
+[source,yaml]
+----
+inputs:
+  - type: fleet-server
+    instrumentation.api_key_file: /path/to/apmAPIKey
+----
++
+It may also be specified by value with the environment variable `ELASTIC_APM_API_KEY`.
+
+APM secret token::
+The APM secret token may be used to gather APM data from {fleet-server}.
++
+It can be directly specified in the instrumentation segment of the configuration:
++
+[source,yaml]
+----
+inputs:
+  - type: fleet-server
+    instrumentation.secret_token: my-apm-secret-token
+----
++
+Or by a file with:
++
+[source,yaml]
+----
+inputs:
+  - type: fleet-server
+    instrumentation.secret_token_file: /path/to/apmSecretToken
+----
++
+It may also be specified by value with the environment variable `ELASTIC_APM_SECRET_TOKEN`.
+
+[[secret-files-guide]]
+== Secret files guide
+
+This guide provides step-by-step examples with best practices on how to deploy secret files directly on a host or through the Kubernetes secrets engine.
+
+[[secret-filesystem]]
+=== Secrets on filesystem
+
+Secret files can be provisioned as plain text files directly on filesystems and referenced or passed through {agent}.
+
+We recommend the following steps in order to improve security.
+
+==== File Permissions
+
+File permissions should not allow for global read permissions.
+
+On MacOS and Linux, file ownership and file permissions can be set with the `chown` and `chmod` commands, respectively.
+{fleet-server} runs as the `root` user on MacOS and Linux, so given a file named `mySecret`, it can be altered with:
+[source,sh]
+----
+sudo chown root:root mySecret # set the user:group to root
+sudo chmod 0600 mySecret      # set only the read/write permission flags for the user, clear group and global permissions.
+----
+
+On Windows, you can use `icacls` to alter the ACL list associated with the file:
+[source,powershell]
+----
+Write-Output -NoNewline SECRET > mySecret          # Create the file mySecret with the contents SECRET
+icacls .\mySecret /inheritance:d                   # Remove inherited permissions from file
+icacls .\mySecret /remove:g BUILTIN\Administrators # Remove Administrators group permissions
+icacls .\mySecret /remove:g $env:UserName          # Remove current user's permissions
+----
+
+==== Temporary filesystem
+
+A temporary filesystem (in RAM) can be used to hold secret files in order to improve security.
+These types of filesystems are normally not included in backups and cleared if the host is reset.
```

Comment:
This statement seems ambiguous.  Do you mean that they are normally not included in backups and WILL BE cleared if the host is reset? 

For clarity, I'd recommend either: 
* These types of filesystems are normally not included in backups and ARE cleared if the host is reset, OR
* These types of filesystems are normally not included in backups and are NOT cleared if the host is reset.

My guess is the first option, but I don't want to guess incorrectly.